src/WebJobs.Script/Workers/Rpc/Configuration/RpcWorkerConfigFactory.cs

```diff
@@ -144,30 +176,16 @@ internal void AddProvider(string workerDir)
 
                     _logger.LogDebug("Found worker config: {workerConfigPath}", workerConfigPath);
 
-                    var workerConfig = GetWorkerConfigJsonElement(workerConfigPath);
```

Comment:
No change in logic for this line. Just moved the method `GetWorkerConfigJsonElement()` to class `WorkerConfigurationHelper`